deps/config/manifests/roles.yml.template

```diff
@@ -1,22 +1,25 @@
-- kind: Role
-  name: anonymous
+- kind: role
   specs:
-    permission:
-      - "read:nodes"
-- kind: Role
-  name: guest
+    name: anonymous
+    permissions:
+      - "proxy:nodes"
+
+- kind: role
   specs:
-    permission:
+    name: guest
+    permissions:
       - "read:*"
-- kind: Role
-  name: signer
+
+- kind: role
```

Comment:
What is the reasoning behind making Kind lowercase? this is another breaking change to be careful of and do it only in case we have strong reasons for it

Reply:
now everything is lowercase